packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -55,13 +55,20 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
   }
 
   private _getActionsDiv(): JSX.Element {
-    if (this.props.actions) {
-      return this.props.isMultiline ?
-        <div className='ms-MessageBar-actions'> { this.props.actions } </div> :
-        <div className='ms-MessageBar-actionsOneline'>
+    if (this.props.isMultiline) {
+      if (this.props.actions) {
+        return <div className='ms-MessageBar-actions'>
+          { this.props.actions }
+        </div>;
+      }
+    }
+    else {
+      if (this.props.actions || (this.props.onDismiss != null)) {
```

Comment:
Minor nit: I think you can just say `(this.props.actions || this.props.onDismiss)`